### DISCUSSION TITLE
Previewing a PDF media conversion, but making the original file downloadable

### DISCUSSION THREAD

**Discussion by amsnvc** [spatie/laravel-medialibrary#3288] [Q&A]
Hey there! I'm trying to display a thumbnail of a PDF file that I made with the Spatie PDF to Image package, but when I want to download said file, I want it to be the PDF, not the preview JPG. Is there a way to work around this? Or is maybe the Pro Library needed? I'm also using Filament PHP as a package. If the Pro Library makes this easier, then please let me know!

**Comment by madurapa** [comment]
If you can put the code you are using it will be easier to answer
